### PR TITLE
Pract 04: Add 01 - 03 | Militsa Toncheva

### PR DESCRIPTION
**Pull request by milicavilica**
Militsa Toncheva
Group 6 2MI0600216

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task01.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+
+using namespace std;
+
+int Sum(int a, int b)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task01.cpp:5]: Use camelCase for naming your functions

File: SI/Sem.04/Pract.04/Task02.cpp [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+
+using namespace std;
+
+bool IsEven(int num)
+{
+	if (num % 2 == 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task02.cpp:7]: num % 2 == 0 is a boolean expression, so it returns a boolean value that you can directly return

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
+using namespace std;
+
+bool IsPositive(int num)
+{
+	if (num > 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:7]: num > 0 is a boolean expression, so it returns a boolean value that you can directly return

File: SI/Sem.04/Pract.04/Task03.cpp [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	}
+	else
+	{
+		return false;
+	}
+}
+
+bool IsNegative(int num)
+{
+	if (num < 0)
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task03.cpp:19]: num < 0 is a boolean expression, so it returns a boolean value that you can directly return

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Some refactoring needed

**Comment by Justsvetoslavov** [comment]
Closing due to merged solutions.
Address comments and contact us if you have questions.

**Pull request closed without merging**